frontend/scripts/react-components/profiles/place-summary.component.jsx

```diff
@@ -74,102 +84,135 @@ class PlaceSummary extends React.PureComponent {
-                    <div className="name -medium">{stateName ? capitalize(stateName) : '-'}</div>
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-        </div>
-
-        <div className="c-overall-info page-break-inside-avoid">
-          <div className="row">
-            <div className="small-12 medium-4 large-3 columns">
-              <div className="c-locator-map map-municipality-banner">
-                {countryName && (
-                  <Map
-                    topoJSONPath={`./vector_layers/municip_states/${countryName.toLowerCase()}/${stateGeoId}.topo.json`}
-                    topoJSONRoot={`${countryName.toUpperCase()}_${stateGeoId}`}
-                    getPolygonClassName={d =>
-                      d.properties.geoid === municipalityGeoId ? '-isCurrent' : ''
-                    }
-                  />
-                )}
-              </div>
-            </div>
+              )}
             <div className="small-12 medium-8 large-9 columns">
               <div className="row">
                 <div className="small-12 columns">
                   <TitleGroup titles={titles} on={onYearChange} />
                 </div>
-                <div className="small-12 columns">
-                  <div className="stat-item">
-                    <div className="legend">area</div>
-                    <div className="value">{areaValue}</div>
-                    <div className="unit">
-                      km<sup>2</sup>
-                    </div>
-                  </div>
-                  <div className="stat-item">
-                    <div className="legend">
-                      {commodityName} land
-                      <HelpTooltip text={get(tooltips, 'profileNode.soyLand')} position="bottom" />
-                    </div>
-                    <div className="value">{soyValue}</div>
-                    <div className="unit">ha</div>
-                  </div>
-                  <div className="stat-item">
-                    <div className="legend">
-                      soy production
-                      <HelpTooltip
-                        text={get(tooltips, 'profileNode.soyProduction')}
-                        position="bottom"
-                      />
-                    </div>
-                    <div className="value">{soyProductionValue}</div>
-                    <div className="unit">t</div>
+                {(areaValue || soyValue || soyProductionValue) && (
+                  <div className="small-12 columns">
+                    {areaValue && (
+                      <div className="stat-item">
+                        <div className="legend">area</div>
+                        <div className="value">{areaValue}</div>
+                        <div className="unit">
+                          km<sup>2</sup>
+                        </div>
+                      </div>
+                    )}
+                    {soyValue && (
```

Comment:
this needs to change right? not always will it be soyValue

Reply:
yes, but the summary will most likely be one of the last things to support multi-context, so I'm not super concerned.